homeassistant/components/stream/fmp4utils.py

```diff
@@ -192,7 +194,7 @@ def read_init(bytes_io: BufferedIOBase) -> bytes:
 )
```

Comment:
It sounds like the `TRANSFORM_MATRIX_TOP` just above here ☝️  should be made a dict keyed with `Orientation`s?

Reply:
Hmm, I thought about that too. That's cleaner, but a tuple index is also more straightforward than a dict lookup especially since we just have ints underneath.
I think I will just add a comment explaining the decision to use a tuple here.

Reply:
Actually the 1st entry is not used either. I acknowledge that it is a little funny but I don't think it's that confusing especially if we add comments. The bigger risk is that there's some mismatch between the indexing and the Enum, but since they're both tied to the EXIF standard (and noted as such) they shouldn't really change.